modules/update-upgrading-cli.adoc

```diff
@@ -46,12 +32,12 @@ $ oc adm upgrade
 ----
 +
 .Example output
-[source,terminal]
+[source,terminal,subs="attributes+"]
```

Comment:
nit: you may no longer need `attributes` now that you no longer use `{product-version}` in the example output?

Reply:
ah - good catch!